lib/game.rb

```diff
@@ -0,0 +1,176 @@
+module PlanetExpress
+  class Game
+
+    def initialize
+      @bender = PlanetExpress::Bender.new(self)
+      @fry = PlanetExpress::Fry.new(self)
+      @hermes = PlanetExpress::Hermes.new(self)
+      @leela = PlanetExpress::Leela.new(self)
+      @zoidberg = PlanetExpress::Zoidberg.new(self)
+      @stable = true
+    end
+
+    def play_game
+      loop do
+        @bender = PlanetExpress::Bender.new(self)
+        @fry = PlanetExpress::Fry.new(self)
+        @hermes = PlanetExpress::Hermes.new(self)
+        @leela = PlanetExpress::Leela.new(self)
+        @zoidberg = PlanetExpress::Zoidberg.new(self)
+        @stable = true
+        do_play_game
```

Comment:
It seems odd that there is a `play_game` and `do_play_game` method. How could you rename for clarity?